Makefile

```diff
@@ -1,8 +1,11 @@
 .PHONY: default all requirements configure repo_sync ironic build ocp_run clean ocp_cleanup host_cleanup
```

Comment:
need to add `prereq` here

Reply:
Yeah but then we'd run them multiple times when common.sh is sourced in multiple scripts?

If we do that we should probably drop a lock file in the `ocp` dir so we can run only once after `make clean`

Reply:
Oops, this is referring to the previous `common.sh` comment

Reply:
I've added the prereq to the common.sh as there were some other prerequisites there.